Test/Tests/RealtimeClientConnectionTests.swift

```diff
@@ -2422,7 +2507,7 @@ class RealtimeClientConnectionTests: XCTestCase {
 
     // RTN15c
```

Comment:
The tests for RTN15c seem to only check that channels in the `ATTACHED` state get re-attached. Per RTN15c6 and RTN15c7, we should be checking the `ATTACHING` and  `SUSPENDED` states, too.

Reply:
[1af28a44aded9df8055c91178845b2e169659078](https://github.com/ably/ably-cocoa/pull/1714#discussion_r1307935811)